src/components/Login.tsx

```diff
@@ -18,18 +18,20 @@ const Login: React.FC = () => {
     <div className={style.login}>
       <h2>Login</h2>
       <form onSubmit={handleLogin}>
-        <TextInput
+        <CustomInput
+          type="text"
           label="E-mail"
           id="email"
           value={email}
           setValue={setEmail}
-        ></TextInput>
-        <TextInput
+        ></CustomInput>
+        <CustomInput
+          type="password"
           label="Password"
           id="password"
           value={password}
           setValue={setPassword}
-        ></TextInput>
+        ></CustomInput>
```

Comment:
If element has no children (is empty f.e. <div></div>), you don't need to have it in pair, you can do `<div/>`.

There is usually a eslint rule for that. But let's start noticing stuff like this

Reply:
eslint rule implemented